src/EditorFeatures/CSharpTest/RemoveUnusedMembers/RemoveUnusedMembersTests.cs

```diff
@@ -32,12 +28,12 @@ public async Task FieldReadViaSuppression()
 #nullable enable
 class MyClass
 {
-    string? [|_field|] = null;
-    void M()
+    string? _field = null;
+    public void M()
     {
         _field!.ToString();
     }
-}", new TestParameters(retainNonFixableDiagnostics: true, parseOptions: new CSharpParseOptions(LanguageVersion.CSharp8)));
```

Comment:
Does the test framework default to latest language version?

Reply:
The test framework defaults to `LanguageVersion.Default`, but we use this instead:

https://github.com/dotnet/roslyn/blob/5d8e2beae1da0c2d789d92b57cac9ff900aa529f/src/EditorFeatures/TestUtilities/CodeActions/CSharpCodeFixVerifier%602%2BTest.cs#L70-L74